test/unit/util_spec.js

```diff
@@ -50,6 +50,90 @@ describe('util', function() {
     });
   });
 
+  describe('isArrayBuffer', function() {
+    it('handles array buffer values', function() {
+      expect(isArrayBuffer(new ArrayBuffer(0))).toEqual(true);
```

Comment:
Since `isArrayBuffer` also works with typed arrays, let's add a test-case using e.g. `Uint8Array` here as well.

Reply:
Done in the new commit.